x-pack/plugins/alerting/common/maintenance_window.ts

```diff
@@ -14,6 +14,13 @@ export enum MaintenanceWindowStatus {
   Archived = 'archived',
 }
 
+export const filterStateStore = {
+  APP_STATE: 'appState',
+  GLOBAL_STATE: 'globalState',
```

Comment:
I still hate to see `globalState`

Reply:
ok fine i will create an issue

Reply:
https://github.com/elastic/kibana/issues/171800